src/test/java/org/mule/extension/file/common/api/matcher/PathMatcherPredicateTest.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extension.file.common.api.matcher;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.regex.PatternSyntaxException;
+
+import org.junit.Test;
+import org.mule.extension.file.common.api.PredicateType;
+
+public class PathMatcherPredicateTest {
+
+  public int failures;
+
+  @Test
+  public void globPattern1() {
+    PathMatcherPredicate pmp = new PathMatcherPredicate("*.txt", PredicateType.EXTERNAL_FILE_SYSTEM, true);
```

Comment:
I understand that there was no test for the old PathMatcherPredicate constructor, right? Please, add more tests with the old constructor and be sure that is working as before.